src/test/java/org/aksw/gerbil/annotator/decorator/ErrorCountingAnnotatorDecoratorTest.java

```diff
@@ -76,7 +76,7 @@ public static class ErrorCausingAnnotatorConfig extends AbstractAdapterConfigura
         private int errorsPerHundred;
 
         public ErrorCausingAnnotatorConfig(int errorsPerHundred) {
-            super("Error causing topic system", false, ExperimentType.ERec);
+            super("Error causing topic system","Un Grouped", false, ExperimentType.ERec);
```

Comment:
Please use the static attribute that you defined further above.

Reply:
Done

Reply:
It doesn't look like that :wink: